packages/client/src/util/CanaryExecutionFactory.ts

```diff
@@ -8,15 +8,15 @@ export default class CanaryExecutionFactory {
           controlScope: {
             scope: '',
             location: '',
-            step: 0,
+            step: 60,
```

Comment:
this is a user friendly change

Reply:
Yeah, I figured we could default it to what the API defaults it to.